bower.json

```diff
@@ -19,22 +19,11 @@
   ],
   "main": "dist/angularfire.js",
   "ignore": [
-    "**/.*",
-    "src",
-    "tests",
-    "node_modules",
-    "bower_components",
-    "firebase.json",
-    "package.json",
-    "Gruntfile.js",
-    "changelog.txt"
+    "**/*",
+    "!dist/angularfire.js"
```

Comment:
you are correct on the minified dist file.

Not sure if README/LICENSE are forced like they are with npm (I made that assumption, but that was probably silly).

Who is consuming `index.js` via bower? I don't think that is needed.